src/homework_oop_v2/Bicycle.java

```diff
@@ -0,0 +1,18 @@
+package homework_oop_v2;
+
+public class Bicycle extends Transport implements Serviceable {
+
+    public Bicycle(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void service() {
+        if (this != null) {
+            System.out.println("Обслуживаем " + this.getModelName());
+            for (int i = 0; i < this.getWheelsCount(); i++) {
```

Comment:
```suggestion
            for (int i = 0; i < getWheelsCount(); i++) {
```
this можно не писать